.codespellrc

```diff
@@ -0,0 +1,4 @@
+[codespell]
+skip = docs/package-lock.json,*.svg
```

Comment:
No reason, no. 

Those files are there just because they threw some error, thus i whitelisted them... There might be some other "special" files we want to ignore too.